cookiecutter.json

```
@@ -2,6 +2,7 @@
   "package_name": "package_name",
   "repository_github_url": "https://github.com/reef-technologies/package-name",
   "is_django_package": "n",
+  "docker_image": "n",
```

```suggestion
  "build_docker_image": "n",
```
suggestion: someone can confuse it to mean the name of the image.